### ISSUE TITLE
"order" or "orders" API, which is correct?

### ISSUE THREAD

**Issue by henn0124** [tradegecko/smaug#2]
The documentation for the Order section the API appears to be incorrect and does not work:

GET https://api.tradegecko.com/order

Instead the requests that do work follow this format:

GET https://api.tradegecko.com/orders

**Comment by lcpriest** [comment]
Good catch!
It was also the case for several other resources:

```
order
purchase_order
invoice
stock_adjustment
stock_transfer
```

It should be updated now.

**Comment by lcpriest** [comment]
Fixed by a5c90d77e2cf9b0909237a7a797860ecc9139957

**Comment by lcpriest** [comment]
Good find @henn0124, thank you!
